### ISSUE TITLE
Package cannot correctly handle 403: Forbidden response from server:

### ISSUE THREAD

**Issue by CountZero1981** [mgufrone/whmcs-api#9]
Package cannot correctly handle 403: Forbidden response from a server:
FatalErrorException in Whmcs.php line 53: Call to undefined method GuzzleHttp\Psr7\Response::json()

**Comment by twhiting9275** [comment]
Yeah, spitting out Guzzle errors here too.
Something's broken

**Comment by N30** [comment]
Comment out lines 61-63 in whmcs.php
```

	public function processResponse($response)
	{
		//if(isset($response['result']) && 'error' === $response['result']
			//|| isset($response['status']) && 'error' === $response['status'] )
			//throw new Exception("WHMCS Response Error : ".$response['message']);
		return json_decode(json_encode($response),'array'===config('whmcs.response', 'object'));
	}

```
There are some issues but it is better than starting from scratch.

**Comment by mgufrone** [comment]
Hai, sorry for late response. Would you like to test the master branch or tag v2.1.0? I've update several code. Guzzle seems like removing json() method, so I should change it into getBody()->getContents()

**Comment by N30** [comment]
Ok I'll try the new version and let you know if I have any issues.

**Comment by MSirous** [comment]
im so sorry but I am a newcomer, in fact i don't know how can i use Whmcs model and i have this problem : 
Class 'App\Http\Controllers\Whmcs' not found
thank you so much for helping me :)

**Comment by mgufrone** [comment]
have you read and follow the docs on the readme at the repo, @MSirous?

**Comment by MSirous** [comment]
yes of course, everything is perfect and i haven't any problem, but i don't know what happen for this class :(
